Sources/BarcodeHeroUI/Controllers/BHCameraScanController.swift

```diff
@@ -6,14 +6,50 @@
     import Foundation
     import UIKit
 
+    #warning("TODO: Make the controller work well in any orientation.")
+
+    @available(iOS 9.0, *)
     open class BHCameraScanController: UIViewController {
-        // MARK: - Properties
+        // MARK: Properties
+
+        private lazy var backgroundView: UIView = {
+            let backgroundView = UIView(frame: UIScreen.main.bounds)
+            backgroundView.backgroundColor = UIColor.black.withAlphaComponent(0.65)
+
+            self.view.addSubview(backgroundView)
+
+            if #available(iOS 11.0, *) {
+                NSLayoutConstraint.activate([
+                    backgroundView.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor),
+                    backgroundView.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor),
+                    backgroundView.topAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.topAnchor),
+                    backgroundView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor),
+                ])
+            } else {
+                NSLayoutConstraint.activate([
+                    backgroundView.leadingAnchor.constraint(equalTo: self.view.leadingAnchor),
+                    backgroundView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor),
+                    backgroundView.topAnchor.constraint(equalTo: self.view.topAnchor),
+                    backgroundView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor),
+                ])
+            }
+
+            return backgroundView
+        }()
+
+        private lazy var crosshairView: BHCrosshairView = {
+            let crosshairView = BHCrosshairView()
+
+            self.view.addSubview(crosshairView)
 
-        @IBOutlet private var backgroundView: UIView!
-        @IBOutlet private var barcodeDataLabel: UILabel!
-        @IBOutlet private var barcodeTypeLabel: UILabel!
-        @IBOutlet private var crosshairImageView: UIImageView!
-        @IBOutlet private var overlayView: UIView!
+            NSLayoutConstraint.activate([
+                crosshairView.centerXAnchor.constraint(equalTo:
+                    self.view.centerXAnchor),
```

Comment:
is this a weird new line? i cant really tell from the diff

Reply:
It is, fixed.